e2e/capacity/transactions.test.ts

```diff
@@ -640,19 +624,17 @@ describe('Capacity Transactions', function () {
       // and is NOT eligible to replenish Capacity, it should error and be dropped
       // from the transaction pool.
       it('fails to pay for a transaction with empty capacity', async function () {
-        const capacityKeys = createKeys('CapKeysEmpty');
-        const capacityProvider = await createMsaAndProvider(fundingSource, capacityKeys, 'CapProvEmpty', FUNDS_AMOUNT);
         const noCapacityKeys = createKeys('noCapacityKeys');
-        const _providerId = await createMsaAndProvider(fundingSource, noCapacityKeys, 'NoCapProvider');
+        const noCapacityProvider = await createMsaAndProvider(fundingSource, noCapacityKeys, 'NoCapProvider');
 
         const delegatorKeys = createKeys('delegatorKeys');
 
         const payload = await generateDelegationPayload({
-          authorizedMsaId: capacityProvider,
+          authorizedMsaId: noCapacityProvider,
           schemaIds: [schemaId],
         });
         const addProviderData = ExtrinsicHelper.api.registry.createType('PalletMsaAddProvider', payload);
-        const grantDelegationOp = ExtrinsicHelper.grantDelegation(
+        const grantDelegationOp = ExtrinsicHelper.createSponsoredAccountWithDelegation(
```

Comment:
The test here doesn't care about what transaction it is doing with capacity, so use the transaction that doesn't have any pre-req.